app/controllers/users/phone_confirmation_controller.rb

```diff
@@ -33,11 +33,12 @@ def assign_phone
       old_phone = current_user.phone
       @updating_existing_number = old_phone.present?
       if @updating_existing_number
-        analytics.track_event('User changed and confirmed their phone number')
+        analytics.track_event('User changed their phone number')
         SmsSenderNumberChangeJob.perform_later(old_phone)
```

Comment:
I put the IdV phone number change notification in there recently, when I refactored the phone confirmation controller into the flow concern. I don't have feelings about continuing to notify users when their verified phone changes. I was just going for symmetry with the MFA confirmation event notification.

I do think it is worthwhile notifying users whenever there is a change in their phone confirmation, as a courtesy and fraud prevention strategy. For IdV I'd be happy to do that after the `Profile` record is activated instead, instead of in the phone confirmation flow. Practically speaking, it's the same request cycle, but a different place in the code. We can address that in a different PR IMO.